cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -278,8 +281,15 @@ def start_kube_dns(sdn_plugin):
     hookenv.log(message)
     hookenv.status_set('maintenance', message)
 
-    context = prepare_sdn_context(sdn_plugin)
```

Comment:
Goodbye hacky old method. I much prefer the new way you build context. Great job here!